### PR TITLE
Remove headers

### PR DESCRIPTION
**Pull request by Artiume**

### CODE REVIEW

**Review by anthonylavado — APPROVED** [review]

File: general/clients/css-customization.md [hunk 1]
```diff
@@ -86,6 +86,32 @@ This will affect the played/watched indicator. Replace the hex color with any va
...
+### Remove Sync Icon from Header
+
+```css
+.headerSyncButton { display: none; }
+```
+
+### Remove User Settings from Header
+
+```css
+.material-icons.person { display: none; }
```
Comment (nielsvanvelzen) [inline general/clients/css-customization.md:112]: I _think_ we use this icon in more places and this CSS will remove those instances. Did you test that?

Comment (Artiume) [reply general/clients/css-customization.md:112]: I didn't see anything abnormal, you have any idea where?

Comment (nielsvanvelzen) [reply general/clients/css-customization.md:112]: At least the menu when you click on your avatar, but I think some places in the dashboard too

Comment (Artiume) [reply general/clients/css-customization.md:112]: All looks good. I didn't see anything on the dashboard

Comment (nielsvanvelzen) [reply general/clients/css-customization.md:112]: image

**Pull request merged**